components/BurgerMenuAndSwitch.js

```diff
@@ -0,0 +1,109 @@
+useless switchmode, just testing*/
+  }
+
+  const [changeDisplay, setChangeDisplay] = useState("none");
+  return (
+    <>
+      <Flex>
+        <Flex position="fixed" top="1rem" right="1rem" align="center">
+          <Flex display={["none", "none", "flex", "flex"]}>
+            <NextLink href="/Home" passHref>
+              <Button as="a" variant="ghost" aria-label="Home" my={5} w="100%">
+                What is happen here?
+              </Button>
+            </NextLink>
+            <NextLink href="/toDOH" passHref>
+              <Button as="a" variant="ghost" aria-label="Home" my={5} w="100%">
+                toDOH
+              </Button>
+            </NextLink>
+            <NextLink href="/toLUV" passHref>
+              <Button as="a" variant="ghost" aria-label="toLUV" my={5} w="100%">
+                toLUV
+              </Button>
+            </NextLink>
+            <NextLink href="/toMEH" passHref>
+              <Button as="a" variant="ghost" aria-label="toMEH" my={5} w="100%">
+                toMEH
+              </Button>
+            </NextLink>
+          </Flex>
+        </Flex>
+        <IconButton
+          color="#FFC12C"
+          bgColor="#499890"
+          aria-label="Open Menu"
+          size="lg"
+          mr={2}
+          icon={<HamburgerIcon />}
+          display={["flex", "flex", "none", "none"]}
+          onClick={() => setChangeDisplay("flex")}
+        />
+
+        {/* <Switch color="green" isChecked={isDark} onChange={toggleColorMode} /> */}
+      </Flex>
+      <Flex
+        color="#FFC12C"
+        width="100vw"
+        bgColor="#499890"
+        zIndex={20}
+        height="100vh"
+        position="fixed"
+        top="0"
+        left="0"
+        overflow="auto"
+        flexDirection="column"
+        display={changeDisplay}
+      >
+        <Flex justify="flex-start">
+          <IconButton
+            bgColor="#499890"
+            marginTop={2}
+            marginRight={2}
+            aria-label="Close Menu"
+            size="lg"
+            icon={<CloseIcon />}
+            onClick={() => setChangeDisplay("none")}
+          />
+        </Flex>
+        <Flex flexDirection="column" align="center">
+          <NextLink href="/Home" passHref>
+            <Button as="a" variant="ghost" aria-label="Home" my={5} w="100%">
+              What is happen here?
+            </Button>
+          </NextLink>
+          <NextLink href="/toDOH" passHref>
+            <Button as="a" variant="ghost" aria-label="Home" my={5} w="100%">
+              toDOH
+            </Button>
+          </NextLink>
+          <NextLink href="/toLUV" passHref>
+            <Button as="a" variant="ghost" aria-label="toLUV" my={5} w="100%">
+              toLUV
+            </Button>
+          </NextLink>
+          <NextLink href="/toMEH" passHref>
+            <Button as="a" variant="ghost" aria-label="toMEH" my={5} w="100%">
+              toMEH
+            </Button>
+          </NextLink>
```

Comment:
I ❤️ the path names

Reply:
Thank you, dear Jan, for your review - Really damn nice written - that motivates!:)  
Chakra UI was suggested to me - during my google research for a relief of responsiveness - by freecodecamp. the responsivness of apps struggeled me from the beginning of the course.( there it is :https://www.freecodecamp.org/news/why-should-you-start-using-chakraui/) I find it quite cool so far, but have long fiddled with it, despite help via youtube . let's see where it still drives me. 
I vow improvement because of prettier - I still thought about formatting, but prettier unfortunately not ;)